src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+    public CEO(String name)
+    {
+        this.name=name;
+    }
+    private String name;
+    private ArrayList<AbstractEmployee> employees_list = new ArrayList();
```

Comment:
In Java the convention is to use camelCase names. So `employees_list` --> `employeeList`